src/main/java/org/hydev/mcpm/SpigotEntry.java

```diff
@@ -0,0 +1,50 @@
+package org.hydev.mcpm;
+
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.plugin.java.JavaPlugin;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Objects;
+import java.util.logging.Logger;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Entrypoint for the Spigot plugin adapter of our program
+ */
+public class SpigotEntry extends JavaPlugin implements CommandExecutor
+{
+    private Logger log;
+
+    /**
+     * onEnable() is called when our plugin is loaded on a Spigot server.
+     */
+    @Override
+    public void onEnable()
+    {
+        // Initialize logger
+        log = getLogger();
+        log.info("[MCPM] Enabled!");
+
+        // Register mcpm command
+        requireNonNull(this.getCommand("mcpm")).setExecutor(this);
```

Comment:
`requireNonNull` without a text argument seems strange here. What's the change of behaviour you're shooting for?